src/main/scala/com/amazon/deequ/checks/Check.scala

```diff
@@ -314,6 +340,22 @@ case class Check(
     hasUniqueness(Seq(column), assertion, hint)
   }
 
+  /**
+   * Creates a constraint that asserts on the uniqueness of a key column.
+   *
+   * @param column          Key column
+   * @param assertion       Function that receives a double input parameter and returns a boolean.
+   *                        Refers to the fraction of unique values.
+   * @param hint            A hint to provide additional context why a constraint could have failed
+   * @param analyzerOptions Options to configure analyzer behavior (NullTreatment, FilteredRow)
+   * @return
+   */
+  def hasUniqueness(column: String, assertion: Double => Boolean, hint: Option[String],
+                    analyzerOptions: Option[AnalyzerOptions])
+  : CheckWithLastConstraintFilterable = {
+    hasUniqueness(Seq(column), assertion, hint, analyzerOptions)
+  }
```

Comment:
Can't we combine all the `hasUniqueness` into one method and update the body of the method based on the existence of parameters?